### DISCUSSION TITLE
Support for OIDC

### DISCUSSION THREAD

**Discussion by rajyraman** [microsoft/powerplatform-actions#533] [General] [answered]
Any idea when OIDC will be supported on Power Platform Actions? For lot of actions, clientid/secret is mandatory. I had to use raw pac to get this working because actions does not recognise the connection using Managed Identity is already active.

![image](https://github.com/microsoft/powerplatform-actions/assets/5035266/8ce88400-d5ad-41e7-8786-04eb08874f62)

![image](https://github.com/microsoft/powerplatform-actions/assets/5035266/3af5fd69-92ef-440e-a607-a6937bece09a)

**Comment by tehcrashxor** [comment, marked as answer]
OIDC support is now out with release [v1.5.0](https://github.com/microsoft/powerplatform-actions/releases/tag/v1.5.0).  See announcement #576
